packages/app/src/index.ts

```diff
@@ -5,15 +5,15 @@ export { load, logger } from "@deliverybot/core";
 const pkg = require(path.join(__dirname, "..", "package.json"));
 export const info: { [k: string]: string } = {
   version: pkg.version,
-  homepage: "https://deliverybot.dev",
-  blog: "https://deliverybot.dev/blog/",
-  integrations: "https://deliverybot.dev/integrations/",
-  status: "https://deliverybot.dev/status/",
-  community: "https://spectrum.chat/deliverybot",
-  documentation: "https://deliverybot.dev/docs/",
-  support: "support@deliverybot.dev",
-  github: "https://github.com/deliverybot/deliverybot",
-  terms: "https://deliverybot.dev/terms/terms/",
-  privacy: "https://deliverybot.dev/terms/privacy/",
-  install: "https://github.com/apps/deliverybot/installations/new",
+  homepage: "https://deliverybot.app.medicmobile.org/",
+  blog: "https://medic.org/stories/",
+  integrations: "https://deliverybot.app.medicmobile.org/integrations/",
+  status: "https://deliverybot.app.medicmobile.org/status/",
+  community: "https://forum.communityhealthtoolkit.org/",
+  documentation: "https://docs.communityhealthtoolkit.org/",
+  support: "app-services@medic.org",
+  github: "https://github.com/medic",
+  terms: "https://twitter.com/Medic",
+  privacy: "https://docs.google.com/document/d/1MaI1rgYMNyCZF2eEjBuvnBDoCYHDKlx4k_N5pkDiWu8/edit",
```

Comment:
Do we have a web-accessible version of this privacy document as opposed to google docs?

Reply:
i forget where i found this... but i think it actually came from medic.org. it's a public read-only url!